frontend/packages/ux-editor/src/components/config/editModal/EditDataModelBinding/EditBinding/EditBindingButtons.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { StudioButton, StudioDeleteButton } from '@studio/components';
+import { XMarkIcon } from '@studio/icons';
+import classes from './EditBinding.module.css';
+import { useTranslation } from 'react-i18next';
+import type { InternalBindingFormat } from '@altinn/ux-editor/utils/dataModelUtils';
+
+export type EditBindingButtons = {
+  handleBindingChange: (binding: InternalBindingFormat) => void;
+  setDataModelSelectVisible: (visible: boolean) => void;
+};
+
+export const EditBindingButtons = ({
+  handleBindingChange,
+  setDataModelSelectVisible,
+}: EditBindingButtons) => {
+  const { t } = useTranslation();
+
+  const handleDelete = () => {
+    const updatedDataModelBinding = {
+      field: '',
+      dataType: undefined,
```

Comment:
Good question, I think the reason was that I wanted to set it back to start when deleting data model binding. 
This is how its config is at default when adding a component to the lage page - see img below.

But I see no problems changing the field to undefined as well when deleted :) 
 
<img width="142" alt="image" src="https://github.com/user-attachments/assets/71f5c9f4-9ced-4a99-bcc5-73f8c49ee058">

Reply:
It would be nice to be consistent at least 🥳 But we should check that it does not confuses the validation in the preview from apps. And that is of course difficult to check considering that the released versions of app-frontend does not support this format yet anyway. But a thing you can do is to run Bjørnars branch of app frontend, with support for the new format, locally, and change the source (in App/views/index.cshtml) in the app in studio to point to the local version. Unless there is actually a RC of app frontend supporting the new format - I am not sure of that!

Or you can ask @bjosttveit  if it will cause issues with there validation 😅

Reply:
He is on holiday 🌴 
After implementing undefined for field when deleting binding (in old format) I get the correct validation message that a binding is needed. I think that is good enough for now, and I can check with Bjørnar when he is back from holiday. What do you think?
Otherways, can you help me testing what you described? 😅

Reply:
I just remember Nina changed the default some while ago to avoid these validations. But there was two originally and after the fix only one was shown, so if we only still show one even though deleting the binding I believe it is fine! 💯